backend/app/graphql/all_users.py

```diff
@@ -0,0 +1,39 @@
+from ..services.implementations.user_service import UserService
+import graphene
+import logging
+from .types import QueryList
+
+
+class UserType(graphene.ObjectType):
+    name = graphene.String()
+    email = graphene.String()
+    role = graphene.String()
+
+
+class AllUsersQuery(QueryList):
+    all_users = graphene.List(
+        UserType,
+        first=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+        role=graphene.String(default_value=""),
+    )
+
+    def resolve_all_users(self, info, first, offset, role):
+        user_service = UserService(logging.getLogger())
```

Comment:
Probably best to just use `services["user_service"]` instead (using `services` exported from `.graphql.services`). That way we can share the same service among all mutations/queries